support-frontend/assets/pages/supporter-plus-landing/twoStepPages/secondStepCheckout.tsx

```diff
@@ -94,13 +92,15 @@ export function SupporterPlusCheckout({
 		</Button>
 	);
 
-	const promotion = useContributionsSelector((state) =>
-		getPromotion(
-			state.page.checkoutForm.product.productPrices,
-			countryId,
-			state.page.checkoutForm.product.billingPeriod,
-		),
-	);
+	const promotion = isSupporterPlus
```

Comment:
Perhaps comment to say `currently promotions can only be applied to recurring S+ products only`